sim/druid/lacerate.go

```diff
@@ -22,6 +23,7 @@ func (druid *Druid) registerLacerateSpell() {
 	druid.Lacerate = druid.RegisterSpell(Bear, core.SpellConfig{
 		ActionID:    core.ActionID{SpellID: 414644},
 		SpellSchool: core.SpellSchoolPhysical,
+		SpellCode:   SpellCode_DruidLacerateDirect,
```

Comment:
DruidLacerateDot is never cast on its own, so SpellCode should probably just be DruidLacerate for this one

Reply:
Yeah this is leftovers from an earlier iteration of the lacerate implementation. Will fix.